### DISCUSSION TITLE
Add legend to menu

### DISCUSSION THREAD

**Discussion by rancidfrog** [iSoron/uhabits#1326] [Feature Requests]
# Checklist
• [x] I am submitting a feature request.
• [x] I have searched for similar issues, but did not find any matches.

# Description

Add 'Legend' to right corner menu, maybe above 'About'.
Legend shows icons used and gives a short description for clarity, 
Mainly it would show what the icons represent:
? 
✓
×
_

# Screenshots
FairEmail implemented a good concise Legend section, although they have more need for it,
![](https://i.imgur.com/bIJDwtD.jpg)

**Comment by rancidfrog** [comment]
I suggested this because I wanted to clarify the icons for skip day and missed habit, as there is nowhere showing which is which.
However,
The description can also explain how each affects the score as well.
If skip days keeps score the same, what happens on ? To the score et ecetera

**Comment by herrbrixter** [comment]
I want this too. I just installed the app but I don't know what the icons mean.
